src/lua/init.c

```diff
@@ -851,6 +851,13 @@ luaopen_tarantool(lua_State *L)
 #endif
 	lua_settable(L, -3);
 
+#ifdef ENABLE_ASAN
+	/* build.asan */
+	lua_pushstring(L, "asan");
+	lua_pushboolean(L, true);
+	lua_settable(L, -3);
+#endif
```

Comment:
Is it a good idea to push explicit `false` if ASAN is disabled?

This way we can distinguish 'unknown' situation (on an old tarantol version) from known-to-disabled on the newer version. Also, it highlight that this value exists in the table for a developer.